programs/client/Client.cpp

```diff
@@ -131,69 +132,64 @@ void Client::showWarnings()
     }
 }
 
-void Client::parseConnectionsCredentials()
+void Client::parseConnectionsCredentials(Poco::Util::AbstractConfiguration & config, const std::string & name)
```

Comment:
```suggestion
void Client::parseConnectionsCredentials(Poco::Util::AbstractConfiguration & config, const std::string & connection_name)
```

Reply:
Yeah, I see your point, but it will conflict with the local variables, there is already `connection_name`, I thought about using `connection`, but there is also class member `connection`, so I went with just `name`, since the name of the function should add enough context to explain `name`.

Reply:
```
const String & connection_name = config.getString(prefix + ".name", "");
```
I'd rather rename it to "name" and use connection_name for the method arg

Reply:
OK, rebased.